index.bs

```diff
@@ -3093,6 +3093,43 @@ This [=registration extension=] and [=authentication extension=] enables use of
                     01           -- Subitem 3: CBOR short for Matcher Protection Type Software
     </pre>
 
+## Authenticator Biometric Criteria Extension (biometricBound) ## {#sctn-authenticator-biometric-criteria-extension}
+
+This [=registration extension=] allows a [=[RP]=] to specify the desired performance of a biometric authenticator
+that could be leveraged when creating the credential. 
+
+: Extension identifier
+:: `biometricBound`
```

Comment:
the extension title is "Authenticator Biometric Criteria", yet the extension identifier is "biometricBound" which is inconsistent.  FAR and FRR are "biometric matching criteria", yes? Perhaps the latter should be "biometricMatchingCriteria" ?   And perhaps the section title ought to be "Biometric Authenticator Matching Criteria" ?